packages/expo-tracking-transparency/src/TrackingTransparency.ts

```diff
@@ -69,6 +75,21 @@ export async function getTrackingPermissionsAsync(): Promise<PermissionResponse>
   return await ExpoTrackingTransparency.getPermissionsAsync();
 }
 
+// @needsAudit
+/**
+ * Check or ask the user to authorize the app and access app-related data that can be used
```

Comment:
let's give this the same description as the methods: https://github.com/expo/expo/pull/13864/files#diff-ef0edece81d91e4ff128fcebc377ce08b51c4b29999acd36fa109b3447dc2be8R20-R30 

and we can take out `needsAudit` then 👍